tests/hipscat/io/conftest.py

```diff
@@ -39,11 +47,63 @@ def assert_text_file_matches(expected_lines, file_name):
                 f"files do not match at line {i+1} " f"(actual: [{contents[i]}] vs expected: [{expected}])"
             )
 
-        metadata_file.close()
-
     return assert_text_file_matches
 
 
+@pytest.fixture
```

Comment:
The cases under `tests` just run on a local file system, so we shouldn't need to replace the directory copy method with our own here.

Reply:
This has been changed